dcrjson/walletsvrcmds.go

```diff
@@ -240,6 +240,33 @@ func NewEstimatePriorityCmd(numBlocks int64) *EstimatePriorityCmd {
 	}
 }
 
+// FundRawTransactionOptions represents the optional inputs to fund
+// a raw transaction.
+type FundRawTransactionOptions struct {
+	ChangeAccount         *string
```

Comment:
The change account is not part of the equivalent bitcoind RPC (instead, any address can be provided as an option).  Why is this different?